backend/api/ches.json

```diff
@@ -2,6 +2,7 @@
   "Ches": {
     "AuthUrl": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token",
     "HostUri": "https://ches-master-9f0fbe-dev.pathfinder.gov.bc.ca/api/v1",
-    "From": "Property Inventory Management System <RealPropertyDivision.Disposals@gov.bc.ca>"
+    "From": "Property Inventory Management System <RealPropertyDivision.Disposals@gov.bc.ca>",
+    "EmailEnabled": true
```

Comment:
Set to `true` so that emails are on, otherwise none will be sent.
Email will currently by default go to the user who activated them.